cypress/e2e/buy-shirt.cy.ts

```diff
@@ -20,14 +20,24 @@ const messageValidateOrderComplete = 'Thank you for your order!'
 
 describe('Buy a black t-shirt', () => {
   it('then the t-shirt should be bought', () => {
+    // Arrange
     loginPage.visitLoginPage()
-    loginPage.signIn(userName, pass)
+    loginPage.setUserName(userName)
+    loginPage.setPassword(pass)
+    loginPage.signIn()
```

Comment:
this change is not necessary, you can preserve the signIn method receiving username and password